### PR TITLE
DOCS-1059, DOCS-1086: Add from Module UI

### PR DESCRIPTION
**Pull request by andf-viam**
Add new UI workflow for adding a service.
 - Also fix existing workflow for adding a component to adapt to all the eng changes.

## Staging

- [/extend/modular-resources/configure/](https://docs-test.viam.dev/475c16525231ca287c580fe1fa112c0a272afec8/public/extend/modular-resources/configure/)

### CODE REVIEW

File: docs/extend/modular-resources/configure.md [hunk 1]
```diff
@@ -8,53 +8,87 @@ description: "Add and configure a module from the Viam Registry on your robot."
...
+
+   {{<imgproc src="extend/modular-resources/configure/add-service-screen.png" resize="400x" declaredimensions=true alt="The add a component modal showing the mlmodelservice triton module pane, with the 'Add module' button shown">}}
+
+When you add a module from the Viam Registry, the custom modular service it provides appears under the **Services** subtab like any other service.
+You can also find [the module itself](#configure-a-module-from-the-viam-registry) listed as **Deployed** under the **Modules** subtab.
+
+If the module requires you to configure specific **Attributes**, click the **URL** link in the [module's configuration pane](#configure-a-module-from-the-viam-registry) to view the specific attribute requirements on the module's GitHub page.
+
+To delete a module added from the Viam Registry, click the trash can icon in the upper-right corner of the module configuration pane in the **Services** tab.
+Deleting a module *does not* delete any configured modular resources it provides.
```
Comment (mcous) [inline docs/extend/modular-resources/configure.md:77]: Good call out. We'll need to update this particular line when deleting the module _does_ delete those resources, though the UI will make it obvious

**Review by mcous — APPROVED** [review]
Screenshots and copy look good to me

File: docs/extend/custom-components-remotes.md [hunk 2]
```diff
@@ -12,10 +12,10 @@ description: "Implement custom components and register them on a server configur
 ---
 
 {{% alert title="Caution" color="caution" %}}
-{{< glossary_tooltip term_id="module" text="Modular resources" >}} are the preferred method of creating custom resource implementations for SDKs with module support unless you are hosting `viam-server` on a non-Linux system or have another issue with compilation.
+[Modular resources](/extend/modular-resources/key-concepts/) are the preferred method of creating custom resource implementations for SDKs with module support unless you are hosting `viam-server` on a non-Linux system or have another issue with compilation.
```
Comment (npentrel) [inline docs/extend/custom-components-remotes.md:15]: Why don't we instead change the glossary entry to link to that 😊

**Review by npentrel — APPROVED** [review]
One optional comment but we can address that later

File: docs/extend/custom-components-remotes.md (hunk 2 above)
Comment (andf-viam) [reply docs/extend/custom-components-remotes.md:15]: Right on, I'll do that tomorrow. 👍

**Pull request merged**